### PR TITLE
Version 0.0.11

### PR DESCRIPTION
**Pull request by larsb24**
- Remove ActionBase
- Drop HandyWindow -> should be decentralised
- Update the Tab design
- Fix Button splash colour
- Add default Tooltip theme
- Remove old TabView
- Update gap
- Revise Checkbox
- Add gitignore
- Fix lints
- Switch to ThemeData instead of ZenitTheme.of
- Add new extensions for ThemeData
- Fix colours
- Add groupValue to ZenitRadioButton

### CODE REVIEW

**Comment by larsb24** [comment]
The main reason for switching from the old immersive TabView to the new ZenitTabBar is that the TabBar and the page content can now be independent from each other, allowing for content in between the TabBar and the page and other layouts.

File: example/lib/pages/tab_view.dart [hunk 1]
```diff
@@ -8,8 +11,65 @@ class ZenitTabViewExample extends StatefulWidget {
 }
 
 class _ZenitTabViewExampleState extends State<ZenitTabViewExample> {
+  late int _selectedIndex;
```
Comment (dvd-bnc) [inline example/lib/pages/tab_view.dart]: you could just do `int _selectedIndex = 0;` so you can remove late and initState

File: lib/src/base/button_base.dart [hunk 2]
```diff
@@ -57,7 +57,6 @@ class _ButtonBaseState extends State<ButtonBase> {
                     color: widget.foregroundColor ?? zenitTheme.foregroundColor,
                   ),
                   child: Align(
```
Comment (dvd-bnc) [inline lib/src/base/button_base.dart]: just use a Center ig?

File: lib/src/layout/tab_view/tab.dart [hunk 3]
```diff
@@ -16,61 +16,67 @@ class ZenitTab extends StatelessWidget {
...
+              shape: RoundedRectangleBorder(
+                borderRadius: kDefaultBorderRadiusMedium,
+                side: BorderSide(color: borderColor),
+              ),
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 8.0),
+                child: Row(
+                  children: [
+                    if (icon != null) SizedBox.square(dimension: 20, child: icon),
+                    const Gap(8),
```
Comment (dvd-bnc) [inline lib/src/layout/tab_view/tab.dart]: i'd recommend dropping the dependency on gap and just using SizedBoxes for gaps like this

**Review by dvd-bnc — COMMENTED** [review]
not half bad! the things i reviewed are just nitpicks, very good job!!

File: lib/src/layout/tab_view/tab.dart (hunk 3 above)
Comment (larsb24) [reply lib/src/layout/tab_view/tab.dart]: I know, you've recommended that a few times already. I'm gonna do it eventually - just not now.

**Pull request merged**